do/pagination.go

```diff
@@ -80,7 +80,7 @@ func PaginateResp(gen Generator) ([]interface{}, error) {
 	}
 
 	// start with second page
-	for i := 1; i < lp; i++ {
+	for i := 2; i <= lp; i++ {
```

Comment:
can you increment `opt.Page++` instead? feels more obvious to me, since the `:=2` here really just means "the page after what we just fetched"